src/package.json

```diff
@@ -74,7 +74,8 @@
   "dependencies": {
     "commander": "^3.0.0-0",
     "fs-extra": "^9.0.1",
-    "gh-pages": "^3.1.0"
+    "gh-pages": "^3.1.0",
+    "jsonc-parser": "^3.0.0"
```

Comment:
Should we install this as a direct dep or assume it as a peer dep? Angular-Devkit uses this as well.